src/index.test.ts

```diff
@@ -24,6 +24,10 @@ test("is Shannon equal to Chanon", () => {
   expect(isChanon("Shannon")).toBe(false);
 });
 
+test("is Shannon equal to Chanon", () => {
```

Comment:
"is Hussein equal to Chanon"

Reply:
Sorry wrote this test while driving a car